src/assets/locales/fr.json

```diff
@@ -217,74 +244,83 @@
   "mic-permissions-modal.title": "Entrer par un appareil",
   "mic-setup-modal.check-mic": "Vérifiez votre micro et l'audio avant d'entrer.",
-  "mic-setup-modal.enter-room-button": "Entrer dans la Salle",
-  "mic-setup-modal.mic-disabled": "Micro Désactivé",
+  "mic-setup-modal.enter-room-button": "Entrer dans la salle",
+  "mic-setup-modal.mic-disabled": "Micro désactivé",
   "mic-setup-modal.mute-mic-toggle": "Couper mon micro",
-  "mic-setup-modal.test-audio": "Cliquer pour Tester l'Audio",
-  "mic-setup-modal.test-mic": "Parler pour Tester le micro",
+  "mic-setup-modal.test-audio": "Cliquer pour tester l'audio",
+  "mic-setup-modal.test-mic": "Parler pour tester le micro",
   "mic-setup-modal.title": "Réglages du micro",
   "more-menu-popover.title": "Plus",
-  "more-menu.close-room": "Fermer la Salle",
+  "more-menu.close-room": "Fermer la salle",
   "more-menu.community": "Communauté",
-  "more-menu.controls": "Controles",
-  "more-menu.create-room": "Créer une Salle",
-  "more-menu.enter-leave-room": "Quitter la Salle",
+  "more-menu.controls": "Commandes",
+  "more-menu.create-room": "Créer une salle",
+  "more-menu.enter-leave-room": "Quitter la salle",
   "more-menu.enter-streamer-mode": "Entrer en mode Streameur",
   "more-menu.exit-streamer-mode": "Sortir du mode Streameur",
-  "more-menu.favorite-room": "Salle Favorite",
-  "more-menu.favorite-rooms": "Salles Favorites",
+  "more-menu.favorite-room": "Salle favorite",
+  "more-menu.favorite-rooms": "Salles favorites",
   "more-menu.help": "Aide",
   "more-menu.invite": "Inviter",
+  "more-menu.preferences": "Préférences",
   "more-menu.privacy": "Notes sur la vie privée",
-  "more-menu.profile": "Changer de Nom & Avatar",
-  "more-menu.report-issue": "Signaler une Erreur",
+  "more-menu.profile": "Changer de nom & avatar",
+  "more-menu.report-issue": "Signaler une erreur",
   "more-menu.room": "Salle",
-  "more-menu.room-info": "Info de la Salle et Réglages",
+  "more-menu.room-info": "Info de la salle et réglages",
   "more-menu.sign-in": "Se connecter",
   "more-menu.sign-out": "Se déconnecter",
   "more-menu.start-tour": "Démarrer la visite guidée",
   "more-menu.support": "Support",
   "more-menu.tos": "Conditions d'utilisation",
   "more-menu.unfavorite-room": "Supprimer la salle des favoris",
   "more-menu.whats-new": "Quoi de neuf",
-  "oauth-screen.description": "Vous devrez vous connecter pour {provider} accéder à la salle.",
+  "more-menu.you": "Vous",
+  "more-menu.you-signed-in-as": "Vous (connecté(e) en tant que {email})",
+  "new-token-scopes.read-rooms.description": "Lire les données de la salle",
+  "new-token-scopes.write-rooms.description": "Écrire les données de la salle",
+  "new-token.back": "Retour",
+  "new-token.generate": "Générer",
+  "new-token.select-scopes-description": "Régler le niveau d'accès que ce jeton aura en choisissant à partir de la liste des portées d'accès.",
```

Comment:
"Régler le niveau d'accès de ce jeton en choisissant dans la liste des portées d'accès."

Reply:
Simpler, I like it.